.env.local

```diff
@@ -1,2 +1,4 @@
-API_URL=http://localhost:3060
-API_KEY=dvl-1510egmf4a23d80342403fb599qd
+WEBSITE_URL=http://localhost:3000
+INVOICES_URL=http://localhost:3002
+API_KEY=94bfa12ba63ba8eaa03e1d591012e5054e8d98fe
+API_URL=https://http://localhost:3060
```

Comment:
Should not be modified. To have a custom local env, create a new `.env` file with your custom variables in it.